test/ssh-connection/mzcompose.py

```diff
@@ -319,15 +319,23 @@ def workflow_rotated_ssh_key_after_restart(c: Composition) -> None:
 def workflow_default(c: Composition) -> None:
     # Test against both standard schema registry
     # and kafka implementations.
-    workflow_basic_ssh_features(c, redpanda=False)
-    workflow_basic_ssh_features(c, redpanda=True)
-    workflow_validate_connection(c)
-    # https://github.com/MaterializeInc/materialize/issues/19252
-    # workflow_kafka_csr_via_ssh_tunnel(c, redpanda=False)
-    # workflow_kafka_csr_via_ssh_tunnel(c, redpanda=True)
```

Comment:
What happened to these lines?

Reply:
And, by the way, the referenced ticket might be resolved (it is closed at least).

Reply:
I enabled the test now and it works, see line 324 in new code.